### ISSUE TITLE
Event selectRange fired when day clicked

### ISSUE THREAD

**Issue by jlainezs** [Paul-DS/bootstrap-year-calendar#240]
selectRange event is fired when a day is clicked, so you got two events in one click:

- selectRange
- clickDay

How to solve this? I expect to got the selectRange event when two days are selected, not when one day is selected.
Thanks!

**Comment by Paul-DS** [comment]
Yes, clicking on a date is considered as selecting a range. If you don't want the both events to be triggered, you can use only the selectRange event, or in the selectRange event listener, check if the startDate and endDate are similar and do nothing in this case.
